scripts/extract_ids/extract_zingzeuids.py

```diff
@@ -0,0 +1,23 @@
+
+import sys
+
+def readfile(filename):
+    with open(filename, 'r') as fin:
```

Comment:
add comments on what this file being read is (how to obtain it and what's the format)